tools/ci_build/build.py

```diff
@@ -1054,6 +1055,8 @@ def generate_build_tree(
         "-Donnxruntime_ARMNN_BN_USE_CPU=" + ("OFF" if args.armnn_bn else "ON"),
         "-Donnxruntime_USE_JSEP=" + ("ON" if args.use_jsep else "OFF"),
         "-Donnxruntime_USE_WEBGPU=" + ("ON" if args.use_webgpu else "OFF"),
+        "-Donnxruntime_ENABLE_PIX_FOR_WEBGPU_EP="
+        + ("ON" if args.enable_pix_capture and args.use_webgpu and is_windows() else "OFF"),
```

Comment:
add check around line 1317 to fail the build explicitly if not building webgpu or not on windows.